piapage/__init__.py

```
@@ -814,50 +807,55 @@ def get_caption_and_background(self):
-                item = items[k]
-                item = item.strip()
-                item = item.replace('\\r','').replace('\\n','') # Fix HTML
-                item = item.strip()
-                if not item: continue
+                new_items = []
+                for k in range(len(items)):
+                    item = items[k]
+                    item = item.strip()
+                    item = item.replace('\\r','').replace('\\n','') # Fix HTML
+                    item = item.strip()
+                    if not item: continue
```

Ruff (0.14.14)

Multiple statements on one line (colon)

(E701)